### PR TITLE
New feature to na_ontap_snapmirror

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Ability to snapmirror between ONTAP and ElementSW
New options
- connection_type
- max_transfer_rate

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
- na_ontap_snapmirror.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/storage/netapp/na_ontap_snapmirror.py [hunk 1]
```diff
@@ -57,23 +64,38 @@
...
      - Optional if this is same as destination password.
-short_description: "NetApp ONTAP Manage SnapMirror"
+  connection_type:
+    description:
+     - Type of SnapMirror relationship.
+     - Pre-requisite for either elementsw_ontap or ontap_elementsw the ElementSW volume should have enableSnapmirror option set to true.
+     - For using ontap_elementsw, elementsw_ontap snapmirror relationship should exist.
+    choices: ['ontap_ontap', 'elementsw_ontap', 'ontap_elementsw']
+    default: ontap_ontap
+    version_added: '2.8'
```
Comment (lonico) [inline lib/ansible/modules/storage/netapp/na_ontap_snapmirror.py]: 2.9

**Review by lonico — APPROVED** [review]
shipit

**Review by lonico — APPROVED** [review]
shipit

**Review by thedoubl3j — APPROVED** [review]
new features documented and tests added. version added fixed. lgtm.

**Pull request merged**